esti/lakectl_local_test.go

```diff
@@ -255,6 +255,38 @@ func TestLakectlLocal_clone(t *testing.T) {
 		require.Contains(t, sanitizedResult, vars["PREFIX"]+"test2.txt")
 		require.NotContains(t, sanitizedResult, vars["PREFIX"]+"nodiff.txt")
 	})
+
+	t.Run("diff with posix permissions", func(t *testing.T) {
+		dataDir, err := os.MkdirTemp(tmpDir, "")
+		require.NoError(t, err)
+		vars["LOCAL_DIR"] = dataDir
+		vars["PREFIX"] = "posix"
+
+		lakectl := LakectlWithPosixPerms()
+		RunCmdAndVerifyContainsText(t, lakectl+" local clone lakefs://"+repoName+"/"+mainBranch+"/"+vars["PREFIX"]+" "+dataDir, false, "Successfully cloned lakefs://${REPO}/${REF}/${PREFIX}/ to ${LOCAL_DIR}.", vars)
+		localVerifyDirContents(t, dataDir, []string{})
+
+		// Add new files to path
+		localCreateTestData(t, vars, []string{
+			vars["PREFIX"] + uri.PathSeparator + "with-diff.txt",
+			vars["PREFIX"] + uri.PathSeparator + "no-diff.txt",
+		})
+
+		res := runCmd(t, lakectl+" local pull "+dataDir, false, false, vars)
+		require.Contains(t, res, "download with-diff.txt")
+		require.Contains(t, res, "download no-diff.txt")
+
+		// the following commit "initializes" the posix permissions for the remote repo
+		runCmd(t, lakectl+" local commit "+dataDir+" --allow-empty-message -m \" \"", false, false, vars)
```

Comment:
nit: make it the commit message instead of a comment
```suggestion
		commitMsg := "initialize the posix permissions for the remote repo"
		runCmd(t, lakectl+" local commit "+dataDir+" --allow-empty-message -m " + commitMsg, false, false, vars)
```

Reply:
TBH I prefer the `fmt.Sprintf` way, I noticed it's like that from previous tests as well

Reply:
I think that `fmt.Sprintf` will make this much less readable.

Reply:
Anyway, good ideas - 
Updated.